woqlclient/woql.py

```diff
@@ -143,21 +143,41 @@ def typecast(self, vara, type, varb):
         return self
 
     def insert(self, node, type, graph=None):
-        """Insert a node with a specific type in a graph
+        """Inserts a new document of a specified type in a graph
```

Comment:
It could also be used other than document, is it not @kevinchekovfeeney ?

Reply:
I've checked in some changes now taking this feedback into account.